Formula/php.rb

```diff
@@ -450,5 +461,57 @@ def plist; <<~EOS
 +    ]) ;;
 +  esac
  ])
-
+ 
+ dnl
+@@ -487,7 +491,11 @@ dnl add an include path.
+ dnl if before is 1, add in the beginning of INCLUDES.
  dnl
+ AC_DEFUN([PHP_ADD_INCLUDE],[
```

Comment:
Can you elaborate on the purpose of this patch ? It looks like MacOS uses extensions that isn't picked up by the configuration script which should be fixed upstream - are there any PRs including this code ?

Reply:
I saw your notice about `Patch changes are to ensure a build can work with system ldap or iconv.` - if we depend on the `openldap` Formula why do we need to patch it so that it builds with system ldap ?

Reply:
Could complement the information about the patches in the initial post and include the links there too ? Just so that its easy to find the explanation if we would need to trace it back at a later date

Reply:
I'm not sure what you're asking for here. Are you suggesting additional URLs be added around the patch line at line 42?

Reply:
No I meant the github post so that we don't have to search through the patch comments =)